app/logic/jwt_authenticator.rb

```diff
@@ -19,7 +19,7 @@ def auth(cookies, params)
 
     def token_from_cookie_or_params(params, cookies)
       if token_from_params(params)
-        JWT.decode(token_from_params(params), Knock.token_public_key, true)
+        JWT.decode(token_from_params(params), Knock.token_public_key, true, algorithms: %w[RS256 HS256])
```

Comment:
the algorithm is in the secret somewhere, right?

Reply:
Yeah, you're right. I had some decoding error and then I changed this line and restarted the server and it worked again, but I think it was the restarting that made it work, not the change of this line.

I'll revert this change.